src/Functions/FunctionsCoding.h

```diff
@@ -954,6 +955,8 @@ class FunctionUUIDStringToNum : public IFunction
 template <typename Impl>
 class EncodeToBinaryRepr : public IFunction
 {
+private:
+    ContextPtr context;
```

Comment:
Where does this context get initialized ? Probably it should be initialized with contructor at 964.
```
static FunctionPtr create(ContextPtr) { return std::make_shared<EncodeToBinaryRepr>(); }
```